models/schema_site_bd.go

```diff
@@ -6,17 +6,20 @@ type SiteBd struct {
 	Value map[string]interface{} `json:",omitempty"`
 }
 
-func NewSchemaSiteBd(ops, path string, anpRef map[string]interface{}, host bool) *SiteBd {
-	var externalepgMap map[string]interface{}
-	externalepgMap = map[string]interface{}{
+func NewSchemaSiteBd(ops, path, mac string, anpRef map[string]interface{}, host bool) *SiteBd {
+	siteBdMap := map[string]interface{}{
 		"bdRef":            anpRef,
 		"hostBasedRouting": host,
 	}
 
+	if mac != "" {
```

Comment:
should anpRef not be bdRef?

```suggestion
func NewSchemaSiteBd(ops, path, mac string, bdRef map[string]interface{}, host bool) *SiteBd {
	siteBdMap := map[string]interface{}{
		"bdRef":            bdRef,
		"hostBasedRouting": host,
	}

	if mac != "" {
```

Reply:
ok